### PR TITLE
Force download of attachments

### PR DESCRIPTION
**Pull request by fanpu**
## Description

Cause attachments to be downloaded instead of being opened with the default MIME handler.

I had to also fix the rubocop-rails dependency in the Gemfile.lock as the previous config broke rubocop for me.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have run rubocop for style check. If you haven't, run `overcommit --install && overcommit --sign` to use pre-commit hook for linting
- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -119,7 +119,6 @@ group :development do
   gem 'binding_of_caller' # enhances better_errors
 
   # static code analyzer
-  gem 'rubocop', require: false
```
Comment (victorhuangwq) [inline Gemfile:122]: you might have accidentally removed rubocop from the gemfile

**Review by xinyis991105 — CHANGES_REQUESTED** [review]
Could you fix Gemfile and Gemfile.lock? Otherwise, LGTM!

**Comment by fanpu** [comment]
> Could you fix Gemfile and Gemfile.lock? Otherwise, LGTM!

Restored the old Gemfiles, previously I was able to have rubocop working with just rubocop-rails and so thought it superseded rubocop

**Review by xinyis991105 — APPROVED** [review]
LGTM!

**Pull request merged**